numpy/linalg/lapack_lite/misc_lite.c.patch

```diff
@@ -0,0 +1,20 @@
+--- a/misc_lite.c
++++ b/misc_lite.c.patched
+@@ -690,7 +690,7 @@ L10:
+ doublereal dlamc3_(doublereal *a, doublereal *b)
+ {
+     /* System generated locals */
+-    doublereal ret_val;
++    volatile doublereal ret_val;
+ 
+ 
+ /*
+@@ -1773,7 +1773,7 @@ L10:
+ doublereal slamc3_(real *a, real *b)
+ {
+     /* System generated locals */
+-    real ret_val;
++    volatile real ret_val;
```

Comment:
This was forgotten in the previous version, so was presumably causing the same bug as the code that the other part of this patch fixed.